server/notebooks/tasks.py

```diff
@@ -0,0 +1,46 @@
+import pytz
+from datetime import datetime, timedelta
+from collections import defaultdict
+from spinach import Tasks
+
+from .models import NotebookRevision
+
+TIME_INTERVAL_IN_MINS = 1
```

Comment:
Let's specify this in seconds, just for a bit more flexibility in the future. I'd also recommend defining this in settings.py, and making it configurable. Maybe call it something like `NOTEBOOK_SPACING_INTERVAL_SECS`

Reply:
Sounds good, thanks.